pep-0514.txt

```diff
@@ -138,11 +163,13 @@ The company name ``PyLauncher`` is reserved for the PEP 397 launcher
 (``py.exe``). It does not follow this convention and should be ignored by tools.
 
 If a string value named ``DisplayName`` exists, it should be used to identify
-the environment category to users. Otherwise, the name of the key should be
-used.
+the environment manufacturer/developer/destributor to users. Otherwise, the name
+of the key should be used. (For ``PythonCore``, the default display name is
+"Python Software Foundation".)
 
 If a string value named ``SupportUrl`` exists, it may be displayed or otherwise
-used to direct users to a web site related to the environment.
+used to direct users to a web site related to the environment. (For
+``PythonCore``, the default support URL is "http://www.python.org/".)
 
```

Comment:
While I have no objection to including SupportURL, I'd like to see an example of the intended use. At the moment, there's little obvious motivation for vendors to include this.